be/src/storage/tablet_manager.cpp

```diff
@@ -142,7 +142,31 @@ Status TabletManager::create_tablet(const TCreateTabletReq& request, std::vector
     int32_t schema_hash = request.tablet_schema.schema_hash;
     LOG(INFO) << "Creating tablet " << tablet_id;
 
-    std::unique_lock wlock(_get_tablets_shard_lock(tablet_id));
+    std::unique_lock wlock(_get_tablets_shard_lock(tablet_id), std::defer_lock);
+    std::unique_lock<std::shared_mutex> base_wlock;
+
+    if (request.__isset.base_tablet_id && request.base_tablet_id > 0) {
```

Comment:
Could you comment when is base tablet specified?

Reply:
Fixed